tests/literals.pyi

```diff
@@ -1,5 +1,13 @@
 from typing import Literal
 
 Literal[None]  # Y061 None inside "Literal[]" expression. Replace with "None"
+Literal[None, None]  # Y061 None inside "Literal[]" expression. Replace with "None"
 Literal[True, None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[True] | None"
 Literal[1, None, "foo", None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[1, 'foo'] | None"
+
+Literal[True, True]  # Y062 Duplicate "Literal[]" member "True"
+Literal[True, True, True]  # Y062 Duplicate "Literal[]" member "True"
+Literal[True, False, True, False]  # Y062 Duplicate "Literal[]" member "True" # Y062 Duplicate "Literal[]" member "False"
+
+# If both Y061 and Y062 would be emitted, only emit Y062
+Literal[None, True, None, True]  # Y062 Duplicate "Literal[]" member "True"
```

Comment:
I did a double-take reading through this again when I saw that Y061 was emitted rather than Y062 for `Literal[None, None]`. I figured it out why that's the best way of doing things, but some more comments would be good here, I think :)

```suggestion
Literal[True, None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[True] | None"

Literal[True, True]  # Y062 Duplicate "Literal[]" member "True"
Literal[True, True, True]  # Y062 Duplicate "Literal[]" member "True"
Literal[True, False, True, False]  # Y062 Duplicate "Literal[]" member "True" # Y062 Duplicate "Literal[]" member "False"

###
# The following rules here are slightly subtle,
# but make sense when it comes to giving the best suggestions to users of flake8-pyi.
###

# If Y061 and Y062 both apply, but all the duplicate members are None,
# only emit Y061...
Literal[None, None]  # Y061 None inside "Literal[]" expression. Replace with "None"
Literal[1, None, "foo", None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[1, 'foo'] | None"

# ... but if Y061 and Y062 both apply
# and there are no None members in the Literal[] slice,
# only emit Y062:
Literal[None, True, None, True]  # Y062 Duplicate "Literal[]" member "True"
```